lib/active_merchant/billing/gateways/paymill.rb

```diff
@@ -47,8 +47,8 @@ def void(authorization, options={})
         commit(:delete, "preauthorizations/#{preauth(authorization)}")
       end
 
-      def store(credit_card, options={})
-        save_card(credit_card)
+        def store(credit_card, money, options={})
```

Comment:
Hmm, this is tricky.

The `#store` method is part of the "official" Gateway abstraction API of ActiveMerchant. Not all gateways implement it, but if they do, they need to adhere to the standard interface. This would now be the only one with a different method signature. 

I don't entirely understand why an amount is needed to store a card, but in lieu of a better alternative, you'll probably have to move it into the `options`.

Also, minor nit, can you fix the indentation here?